easybuild/easyconfigs/p/PETSc/PETSc-3.12.3-foss-2019b.eb

```diff
@@ -0,0 +1,40 @@
+name = 'PETSc'
+version = '3.12.3'
+
+homepage = 'https://www.mcs.anl.gov/petsc'
+description = """PETSc, pronounced PET-see (the S is silent), is a suite of data structures and routines for the
+ scalable (parallel) solution of scientific applications modeled by partial differential equations."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+# https:// does not work here
+source_urls = ['ftp://ftp.mcs.anl.gov/pub/petsc/release-snapshots']
+sources = [SOURCELOWER_TAR_GZ]
+patches = [
+    'PETSc_ranlib-fix.patch',
+]
+checksums = [
+    '91f77d7b0f54056f085b9e27938922db3d9bb1734a2e2a6d26f43d3e6c0cf631',  # petsc-3.12.3.tar.gz
+    '64cf9d5008d5e92117e65bdec5316d991b6a6b8c8ecf7ea46eb790a498266297',  # PETSc_ranlib-fix.patch
+]
+
+builddependencies = [('CMake', '3.15.3')]
+
+dependencies = [
+    ('Boost', '1.71.0'),
+    ('METIS', '5.1.0'),
+    ('SCOTCH', '6.0.9'),
+    ('SuiteSparse', '5.6.0', '-METIS-5.1.0'),
+    ('Hypre', '2.18.2'),
+]
+
+configopts = '--LIBS="$LIBS -lrt"'
+
+shared_libs = 1
+
+# only required when building PETSc in a SLURM job environment
+# configopts = '--with-batch=1 --known-mpi-shared-libraries=1 --known-64-bit-blas-indices=0 '
```

Comment:
```diff
-configopts = '--LIBS="$LIBS -lrt" '
+configopts = '--LIBS="$LIBS -lrt"'
 
 # only required when building PETSc in a SLURM job environment
-# configopts += '--with-batch=1 --known-mpi-shared-libraries=1 --known-64-bit-blas-indices=0 '
+# configopts = '--with-batch=1 --known-mpi-shared-libraries=1 --known-64-bit-blas-indices=0 '
```

why change this? (relative to PETSc-3.11.1-foss-2019a.eb) As it is, if one simply uncomments this line for SLURM, it will override the `configopts` above

Reply:
Corrected in latest commit. Thanks